src/articles/index.tsx

```diff
@@ -15,25 +16,94 @@ import { Header } from '../misc/header';
 
 export function Articles(_: unknown, renderer: RendererLike<Node>) {
   return <>
-    <Route path='**/unapproved' comp={() =>
-      <ArticleList title='Unapproved Articles'
-        articles={fromPromise(getUnapprovedArticles(authToken()!))}
+    <Route path='**/unapproved' comp={() => {
```

Comment:
this is a huge inline code, which is basically duplicated for unapproved articles as well. could you please extract this into its own component?

Reply:
Sure thing